gandalf/src/main/java/io/github/btkelly/gandalf/Gandalf.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2016 Bryan Kelly
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.github.btkelly.gandalf;
+
+/**
+ * This is the public api that allows consumers to configure the Gandalf library.
+ */
+public class Gandalf {
```

Comment:
I figured that would be taken care of with the create builder issue #10. I just needed to put something in so I could use it in the api.